docs-ui/stories/components/replays/replayPlayer.stories.js

```diff
@@ -8,14 +10,32 @@ export default {
   component: ReplayPlayer,
 };
 
+const ManualResize = styled('div')`
+  resize: both;
+  overflow: auto;
+  width: 50%;
+`;
+
 export const ScaledReplayPlayer = () => (
   <ReplayContextProvider events={events}>
-    <ReplayPlayer />
+    <ManualResize>
+      <ReplayPlayer />
+    </ManualResize>
```

Comment:
Are these non-buffer-related changes?

Reply:
I wrapped all the examples with `<ManualResize>` mainly because the `width: 50%` guarantees that the whole player will be visible inside the storybook pane without scrolling.

It made my life easier to play with the examples because I can see that "fast-forward" badge in the lower-left corner without scrolling, and can resize the player without resizing the whole window.